tests/operators/test_update.py

```diff
@@ -162,6 +162,38 @@ def test_adj_op(
     npt.assert_allclose(image_changed, image_true, atol=1e-3, rtol=1e-3)
 
 
+@param_array_interface
+def test_adj_op_density(
+    operator,
+    array_interface,
+    kspace_data,
+):
+    """Test the batch type 1 (adjoint)."""
+    kspace_data = to_interface(kspace_data, array_interface)
+    jitter = np.random.rand(operator.samples.shape[0]).astype(np.float32)
+    if operator.backend == "cufinufft":
+        jitter = to_interface(jitter, "cupy")
```

Comment:
This kind of patch should rather be directly implemented in code. If the user update cufinufft density it is updated as a cupy vector internally (or we raise an error)

Reply:
Lol yes.